src/domains/wallet-common/03_cosmosdb.tf

```diff
@@ -74,3 +74,124 @@ resource "azurerm_cosmosdb_mongo_database" "wallet" {
 }
 
 # Collections
+locals {
+  collections = [
+    {
+      name = "services"
+      indexes = [{
+        keys   = ["_id"]
+        unique = true
+        },
+        {
+        keys   = ["name"]
+        unique = true
+        }
+      ]
+      shard_key = null
+    },
+    {
+      name = "wallets"
+      indexes = [{
+          keys   = ["_id"]
+          unique = true
+        },
+        {
+          keys   = ["userId"]
+          unique = false
+        }
+      ]
+      shard_key = "_id"
+    },
+    {
+      name = "wallet-log-events"
+      indexes = [{
+          keys   = ["_id"]
+          unique = true
+        },
+        {
+          keys   = ["walletId", "eventType", "timestamp"]
+          unique = true
+        },
+        {
+          keys   = ["walletId"]
```

Comment:
this index should be already covered by the above one, so maybe it's useless. What do you think?

Reply:
We updated the indexes composition and definition. Thank you